internal/distro/rhel90/distro.go

```diff
@@ -95,7 +95,7 @@ var distroMap = map[string]distribution{
 		vendor:             "redhat",
 		ostreeRefTmpl:      "rhel/9/%s/edge",
 		isolabelTmpl:       "RHEL-9-1-0-BaseOS-%s",
-		runner:             "org.osbuild.rhel91",
```

Comment:
This one should not change.

Reply:
Then this commit will be empty, right?

Reply:
yes, it should have been dropped from the PR 😉